### ISSUE TITLE
TypeError: Cannot assign to read only property 'map' of object '#<QueryCursor>'

### ISSUE THREAD

**Issue by MRX-0101** [Automattic/mongoose#11384]
when i run the script it's come this error 

```C:\Users\MRX\Downloads\Groove-main\Groove-main\node_modules\mongoose\lib\cursor\QueryCursor.js:144
QueryCursor.prototype.map = function(fn) {
                          ^

TypeError: Cannot assign to read only property 'map' of object '#<QueryCursor>'
    at Object.<anonymous> (C:\Users\MRX\Downloads\Groove-main\Groove-main\node_modules\mongoose\lib\cursor\QueryCursor.js:144:27)
    at Module._compile (node:internal/modules/cjs/loader:1097:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1151:10)
    at Module.load (node:internal/modules/cjs/loader:975:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:999:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (C:\Users\MRX\Downloads\Groove-main\Groove-main\node_modules\mongoose\lib\query.js:12:21)
    at Module._compile (node:internal/modules/cjs/loader:1097:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1151:10)```

**Comment by Grayson-code** [comment]
Same Error Has been happening to me its particularly coming from the core files ig,

Average day of a backend dev :/

**Comment by Uzlopak** [comment]
@vkarpov15 

This can be closed.

**Comment by Uzlopak** [comment]
This is resolved with 6.2.2
